1.1/translations/funcsrules_de.ttl

```diff
@@ -0,0 +1,453 @@
+    skos:prefLabel "GeoSPARQL Funktionen"@de .
+
+funcs:eh a skos:Collection ;
+    skos:definition """Alle GeoSPARQL Egenhofer Topologischen Funktionen, die in der GeoSPARQL 1.1 Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL Egenhofer Topologische Funktionen"@de .
+
+funcs:rcc a skos:Collection ;
+    skos:definition """Alle in der GeoSPARQL 1.1 Spezifikation definierten GeoSPARQL RCC8 Topologischen Funktionen"""@de ;
+    skos:prefLabel "GeoSPARQL RCC8 Topologische Funktionen"@de .
+
+funcs:sf a skos:Collection ;
+    skos:definition """Alle GeoSPARQL Simple Features Topologische Funktionen, die in der GeoSPARQL 1.1 Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL Simple Features Funktionen"@de .
+
+funcs:nonTopo a skos:Collection ;
+    skos:definition """Alle nicht-topologischen GeoSPARQL Query Funktionen, die in der GeoSPARQL 1.1-Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL Nicht-topologische Query Funktionen"@de .
+
+funcs:common a skos:Collection ;
+    skos:definition """Alle GeoSPARQL Common Query Funktionen, die in der GeoSPARQL 1.1-Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL Common Query Funktionen"@de .    
+
+funcs:sa a skos:Collection ;
+    skos:prefLabel "GeoSPARQL Raumbezogene Aggregatfunktionen"@de .
+
+rules: a skos:Collection ;
+    skos:definition """Alle GeoSPARQL RIF Rules, die in der GeoSPARQL 1.1-Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL RIF Rules"@de .
+
+rules:eh a skos:Collection ;
+    skos:definition """Alle GeoSPARQL Egenhofer RIF Rules, die in der GeoSPARQL 1.1 Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL Egenhofer Topologische RIF Rules"@de .
+
+rules:rcc a skos:Collection ;
+    skos:definition """Alle GeoSPARQL RCC8 RIF Rules, die in der GeoSPARQL 1.1 Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL RCC8 Topologische RIF Rules"@de .
+
+rules:sf a skos:Collection ;
+    skos:definition """Alle GeoSPARQL Simple Features Topological Rules, die in der GeoSPARQL 1.1 Spezifikation definiert sind"""@de ;
+    skos:prefLabel "GeoSPARQL Simple Features Rules" .
+
+funcs:area a skos:Concept ;
+    skos:definition "Gibt die Fläche einer Geometrie in Quadratmetern zurück."@de ;
+    skos:prefLabel "area"@de .
+
+funcs:asDGGS a skos:Concept ;
+    skos:definition "Konvertiert eine gegebene Geometrie in eine äquivalente DGGS-Darstellung, formuliert nach dem spezifischen DGGS-Literal, das mit dem Parameter specificDggsDatatype angegeben wird."@de ;
+    skos:prefLabel "asDGGS"@de .
+	
+funcs:asGeoJSON a skos:Concept ;
+    skos:definition "Konvertiert eine gegebene Geometrie in eine äquivalente GeoJSON-Darstellung. Koordinaten werden in das CRS84-Koordinatensystem konvertiert. Dies ist das einzige gültige Koordinatensystem, das in einem GeoJSON-Literal verwendet werden kann."@de ;
+    skos:prefLabel "asGeoJSON"@de .
```

Comment:
I think all of these prefLabels should be changes from being exactly like the IRI - `asGeoJSON` to being rendered in natural language. In the English funcsrules.ttl this would be "as GeoJSON". I'll make a PR for that change in the English version and I think you should do the same here!

Reply:
I was reluctant to do that because we describe an actual function name here.
People should identify the function based on its name right?
Do you really think it makes sense to use the German translation of a function name which in its translated form can not be accessed in SPARQL?

Reply:
That is a good point. I suppose some of the function names are acronyms and so on anyway, e.g. rcc8...

Let's make our lives easier then and make no change here, unless someone else raises the issue!